tensorflow-framework/tensorflow-data.md

```diff
@@ -0,0 +1,201 @@
+Tensorflow Data (Java)
```

Comment:
I'm still not sure if we should have individual `.md` files per package or just one big `README.md` that is correctly split into sections... but I think that's fine for now.